### ISSUE TITLE
Support aws_internet_gateway

### ISSUE THREAD

**Issue by eliecharra** [cloudskiff/driftctl#98]
**Description**
Add support for aws_internet_gateway

**Comment by sjourdan** [comment]
Using the following simple tf code, I could not detect a drift I manually generated (and that TF could see) 

```
resource "aws_internet_gateway" "staging_gw" {
  vpc_id = aws_vpc.staging.id

  tags = {
    Name = "Staging Internet Gateway"
    Terraform = "true"
  }
}
``` 

If I go to the was console and detach this internet gateway from its VPC, it's not detected by driftctl (`vpc_id` field) but it is by terraform: 

```
  # aws_internet_gateway.staging_gw will be updated in-place
  ~ resource "aws_internet_gateway" "staging_gw" {
        id       = "igw-05a657a54c849474d"
        tags     = {
            "Name"      = "Staging Internet Gateway"
            "Terraform" = "true"
        }
      + vpc_id   = "vpc-056a722ce71168568"
        # (2 unchanged attributes hidden)
    }
```

Using 1f66e51

**Comment by wbeuil** [comment]
I don't see any problem here, I created a vpc and an internet gateway via terraform as per the code below 

```
resource "aws_vpc" "vpc" {
  cidr_block = "10.0.0.0/16"
}

resource "aws_internet_gateway" "main" {
  vpc_id = aws_vpc.vpc.id
  tags = {
    Name = "main"
  }
}

```

I ran on time driftctl, it showed that all was good. Then I detach via the console the internet gateway from the vpc. I ran again driftctl and this is the output.

![image](https://user-images.githubusercontent.com/8110579/105400715-9c414280-5c25-11eb-96b6-004beda8572b.png)

It detected the drift as expected.
